Helm/opensearch-dashboards/values.yaml

```diff
@@ -59,22 +58,32 @@ securityContext:
   runAsNonRoot: true
   runAsUser: 1000
 
-
 config:
   dashboards.yml:
```

Comment:
FYI : This config is going to be changed via #38 . Some instances of `dashboards` will get replaced with `opensearch-dashboards` and `opensearch_dashboards` respectively.